server/utils/workflow-util.js

```diff
@@ -180,7 +181,7 @@ exports.notifyWorkflowExecutionStatus = async ({hpccURL,executionStatus,dataflow
                         <p> ${success_message } </p>
                         <p> Hello, </p>
                         <p>Successfully executed ${dataflowName} on ${clusterName} </p>
-                        <p> To view workflow execution details in Tombolo please click <a href="${process.env.WEB_URL}/${appId}/dataflowinstances/dataflowInstanceDetails/${dataflowId}/${jobExecutionGroupId}"> here </a>
+                        <p> To view workflow execution details in Tombolo please click <a href="${process.env.WEB_URL}${appId}/dataflowinstances/dataflowInstanceDetails/${dataflowId}/${jobExecutionGroupId}"> here </a>
```

Comment:
Will this URL fail if process.env.WEB_URL contains a trailing slash? Should it be handled in the code itself in case someone mistakenly adds a trailing slash? Even better to completely avoid this property and just use "href="/${appId}/dataflowinstances/dataflowInstanceDetails/${dataflowId}/${jobExecutionGroupId}""? Worth checking whether it will work?

Reply:
Yes it seems to be failing because the process.env.WEB_URL already has a '/'  and I was adding on code also. We can get rid of '/' from the env and add only on the code. Regarding getting rid of the property, I did not find a way.